config/i18n-tasks.yml

```diff
@@ -102,6 +102,7 @@ ignore_unused:
   - 'date.formats.*'
   - 'general.*'
   - 'errors.*'
+  - 'views.public_pages.maybe_ineligible.diy_criteria_list.{item_1,item_2,item_3,item_4}'
```

Comment:
Is there a reason we should ignore these?

Reply:
@bengolder I was trying to avoid duplication in the view by iterating [here](https://github.com/codeforamerica/vita-min/pull/396/files#diff-d23db4b6c12dc20c355b3387ac27349eR28). Unfortunately i18n-tasks is not clever enough to see that those interpolated keys are being used so the spec fails. I am happy to just be explicit in the view if you would prefer to avoid having this ignored in the yml?

Reply:
explicit in the view sounds preferable to me